netdata-installer.sh

```diff
@@ -314,7 +314,7 @@ do
 
 	count=$((count + 1))
 
-	pid=$(cat /var/run/netdata/netdata.pid 2>/dev/null)
+	pid=$(cat /var/run/netdata/netdata.pid 2>/dev/null || cat /var/run/netdata.pid 2>/dev/null)
```

Comment:
why do you need both? what do you need it anyway? both openrc and systemd will not use it, so you cannot assume anything if missing.